### PR TITLE
[DOCS-5447] Bryce/add azure logging guide

### PR DESCRIPTION
**Pull request by buraizu**
### What does this PR do?
Moves the Azure logging instructions from the main Azure doc into a separate guide in the logs folder

### Motivation
DOCS-5413

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/logs/guide/azure-logging-guide.md [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+{{% /tab %}}
+{{< /tabs >}}
+
+## Log Archiving
+
+Archiving logs to Azure Blob Storage requires an App Registration even if you are using the Azure Native integration. To archive logs to Azure Blob Storage, follow the setup instructions to configure the integration using an App Registration. App Registrations created for archiving purposes do not need the `Monitoring Reader` role assigned.
+
+Once you have an App Registration configured, you can [create a log archive][62] that writes to Azure Blob Storage. 
+
+**Note**: If your storage bucket is in a subscription being monitored through the Azure Native integration, a warning is displayed in the Azure Integration Tile about the App Registration being redundant. You can ignore this warning.
```
Comment (maycmlee) [inline content/en/logs/guide/azure-logging-guide.md:303]:
```suggestion
**Note**: If your storage bucket is in a subscription being monitored through the Azure Native integration, a warning is displayed in the Azure Integration tile about the App Registration being redundant. You can ignore this warning.
```

File: content/en/logs/guide/azure-logging-guide.md [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+[6]: https://docs.microsoft.com/en-us/learn/modules/store-app-data-with-azure-blob-storage/
+[7]: https://docs.microsoft.com/en-us/azure/azure-functions/functions-create-first-azure-function
+[8]: https://github.com/DataDog/datadog-serverless-functions/blob/master/azure/blobs_logs_monitoring/index.js
+[9]: https://app.datadoghq.com/organization-settings/api-keys
+[10]: https://app.datadoghq.com/logs
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Log Archiving
```
Comment (maycmlee) [inline content/en/logs/guide/azure-logging-guide.md:297]:
```suggestion
## Log archiving
```

File: content/en/logs/guide/azure-logging-guide.md [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+
+[1]: /integrations/guide/azure-troubleshooting/#automated-log-collection
+{{% /tab %}}
+
+{{% tab "Manual installation" %}}
+
+To send logs from Azure to Datadog, follow this general process:
+
+1. Create an [Azure Event Hub][1].
+2. Setup the Datadog-Azure [function with an Event hub trigger][2] to forward logs to Datadog.
```
Comment (maycmlee) [inline content/en/logs/guide/azure-logging-guide.md:157]:
```suggestion
2. Set up the Datadog-Azure [function with an Event hub trigger][2] to forward logs to Datadog.
```

File: content/en/logs/guide/azure-logging-guide.md [hunk 4]
```diff
@@ -0,0 +1,325 @@
...
+
+## Azure standard integration log collection
+
+Datadog recommends submitting logs from Azure to Datadog with the Agent or DaemonSet. For some resources it may not be possible. In these cases, you can create a log forwarding pipeline using an Azure Event Hub to collect [Azure Platform Logs][57]. For resources that cannot stream Azure Platform Logs to an Event Hub, you can use the Blob Storage forwarding option.
+
+{{< tabs >}}
+
+{{% tab "Automated Installation" %}}
+
+To get started, click the button below and fill in the form on Azure Portal. The Azure resources required to get activity logs streaming into your Datadog account will be deployed for you.
```
Comment (maycmlee) [inline content/en/logs/guide/azure-logging-guide.md:62]:
```suggestion
To get started, click the button below and fill in the form on Azure Portal. The Azure resources required to get activity logs streaming into your Datadog account are deployed for you.
```

File: content/en/logs/guide/azure-logging-guide.md [hunk 5]
```diff
@@ -0,0 +1,325 @@
...
+
+{{% tab "Manual installation" %}}
+
+To send logs from Azure to Datadog, follow this general process:
+
+1. Create an [Azure Event Hub][1].
+2. Setup the Datadog-Azure [function with an Event hub trigger][2] to forward logs to Datadog.
+3. Configure your Azure services to stream logs to the Event Hub by creating a [diagnostic setting][3].
+
+The instructions below walk through a basic, initial setup using the Azure Portal. All of these steps can be performed with the CLI, Powershell, or resource templates by referring to the Azure documentation.
```
Comment (maycmlee) [inline content/en/logs/guide/azure-logging-guide.md:160]:
```suggestion
The instructions below walk through a basic, initial setup using the Azure portal. All of these steps can be performed with the CLI, Powershell, or resource templates by referring to the Azure documentation.
```

**Review by maycmlee — COMMENTED** [review]
Just a couple of small nits! Also adding a comment in the Jira card.

File: content/en/logs/guide/azure-logging-guide.md (hunk 4 above)
Comment (buraizu) [reply content/en/logs/guide/azure-logging-guide.md]: Thanks for catching this!

**Comment by buraizu** [comment]
Thanks for the review @maycmlee. I've incorporated your feedback, and also split the US3 information into a separate guide (content hasn't been changed though).

**Review by maycmlee — APPROVED** [review]
LGTM!

File: content/en/logs/guide/azure-logging-guide.md [hunk 6]
```diff
@@ -0,0 +1,275 @@
...
+
+{{% tab "Blob Storage" %}}
+
+To collect logs from all of your Azure App Services, follow this general process:
+
+1. Set up [Azure Blob Storage][1] from the [Azure portal][2], [Azure Storage Explorer][3], [Azure CLI][4], or [Powershell][5].
+2. Set up the [Datadog-Azure Function](#create-a-new-azure-blob-storage-function) which forwards logs from your blob storage to Datadog.
+3. Configure your Azure App Services to [forward their logs to the Blob Storage][6].
+
+#### Create a new Azure Blob Storage function
```
Comment (claudiadadamo) [inline content/en/logs/guide/azure-logging-guide.md:226]: IMO we should add a note that this is not the suggested option for log forwarding to make it clear that we want customers to use eventhubs instead

Comment (buraizu) [reply content/en/logs/guide/azure-logging-guide.md:226]: Thanks for clarifying that -- I added a note to this effect, but let me know if you have any feedback on it.

**Comment by buraizu** [comment]
Received approval from Azure team via Slack.

**Pull request merged**